content/projects/gsoc/2019/index.adoc

```diff
@@ -25,10 +38,7 @@ GSoC Organization admins in 2019:
 * link:https://github.com/lloydchang[Lloyd Chang]
 * link:https://github.com/oleg-nenashev/[Oleg Nenashev]
 
-=== Links
-
-* link:/projects/gsoc/2019/schedule[Jenkins year-round GSoC schedule]
-* link:/projects/gsoc/gsoc2018-project-ideas[GSoC 2018 Project ideas]
-* link:/blog/2018/01/06/gsoc2018-call-for-mentors[Call for Mentors]
-* link:https://summerofcode.withgoogle.com/organizations/5572716199936000/[Jenkins page on GSoC 2018 website]
-
+Jenkins Org Admins mailing list: jenkins-gsoc-2019-orgs@googlegroups.com.
```

Comment:
The correct email address is jenkins-gsoc-2019-org-admins@googlegroups.com

Commit c2bd350a7ed36192063ed44ddd30bf4d1a995b0f lists an incorrect email address; I received following error when I tried it:
> Address not found
> Your message wasn't delivered to jenkins-gsoc-2019-orgs@googlegroups.com because the address couldn't be found, or is unable to receive mail.